_plugins/translations.rb

```diff
@@ -0,0 +1,21 @@
+module Jekyll
+  module TranslationsFilter
+    # sorts the given array by their translated value
+    def sort_translated(keys, prefix = '')
+      keys.sort_by do |key|
+        translate(key, prefix)
+      end
+    end
+
+    def translate(key, prefix)
+      locale = @context.registers[:page]['lang']
+      data = @context.registers[:site].data
+
+      data.dig(locale, *prefix.split('.'), key) ||
+        data.dig('en', *prefix.split('.'), key) ||
```

Comment:
It doesn't sit well with me that we'd allow silently falling back to English, since it could very easily be a mistake. We do have a separate test to check for this, which is currently failing 😅 

Personally, I don't see much of an issue with letting the duplicate texts exist in non-English locale settings if that's how it's meant to be shown in that language.

That being said, I could still see this function being useful:

- Raising an error on a missing key could help prevent mistakes.
- Separately I'd been wanting to find a way to automate "are all the settings keys being used". This might make that easier? Not sure.

Reply:
yeah we could raise an error, or do something like return the string "missing translation x.y.z" like Rails does in development mode?

Reply:
this whole file is removed for now, so we can come back and add a translate filter/tag later if we want to